.travis.yml

```diff
@@ -7,6 +7,6 @@ install:
 script:
   - "pep8 ."
   - "pylint --load-plugins pylint_django volontulo"
-  - "py.test -v --cov . --cov-report term-missing"
+  - "py.test --ds=volontulo.pytest_settings -v --cov . --cov-report term-missing"
```

Comment:
This -ds looks suspiciously for me. 
py.test has -s that is shortcut for --capture=no and -c to load specified configuration from file, probably volontulo.pytest_settings.py.